scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -961,19 +918,18 @@ BEGIN
   -- Put the primary key sequence in the right schema
   -- If the new table is not moving, better ensure the sequence name
   -- is unique
-  destseq := relname || '_' || const.pkey || '_seq';
-  destseq := _CDB_Unique_Relation_Name(destschema, destseq);
+  destseq := cartodb._CDB_Unique_Identifier(NULL, relname, '_' || const.pkey || '_seq', destschema);
   destseq := Format('%I.%I', destschema, destseq);
   PERFORM _CDB_SQL(Format('CREATE SEQUENCE %s', destseq), '_CDB_Rewrite_Table');
 
   -- Salt a temporary table name if we are re-writing in place
   -- Note copyname is already escaped and safe to use as identifier
   IF destschema = relschema THEN
-    copyname := Format('%I.%I', destschema, Format('%s_%s', destname, salt));
+    copyname := Format('%I.%I', destschema, cartodb._CDB_Unique_Identifier(NULL, destname, '_' || salt), destschema);
```

Comment:
I would remove the salting thing, it was done just and precisely to get unique names, apparently (sorry I did not notice before).